storage/engine/mvcc.go

```diff
@@ -1634,9 +1634,16 @@ func MVCCIterate(ctx context.Context,
 				}
 				break
 			}
-
 		} else {
-			iter.Seek(MakeMVCCMetadataKey(metaKey.Key.Next()))
+			if buf.meta.IsInline() {
```

Comment:
Raft log entries are inline values.

On Wed, Apr 27, 2016 at 12:04 PM, Spencer Kimball <notifications@github.com>
wrote:

> In storage/engine/mvcc.go
> <https://github.com/cockroachdb/cockroach/pull/6320#discussion_r61286305>:
>
> >  		} else {
> > -			iter.Seek(MakeMVCCMetadataKey(metaKey.Key.Next()))
> > +			if buf.meta.IsInline() {
>
> Why does this make such a big difference in this benchmark's speed? The
> data being written in the bench test looks to be non-inline...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/6320/files/7e6d02dd40c2613e250f3cb07279a80518b68dd1#r61286305>
>